src/vectorchord/advanced-features/pre-and-post-filtering.md

```diff
@@ -0,0 +1,44 @@
+# Pre And Post Filtering <badge type="tip" text="since v0.4.0" />
+
+In VectorChord, the most costly step in a vector search is `rerank`, which computes vector distances precisely.
+
+For a vector search with a filter, there are two ways to handle the interaction of filter and rerank:
+
+Pre-filtering: Apply the filter before the rerank step.
+
+$$search \Rightarrow filter \Rightarrow rerank \longmapsto candidates$$
+
+Post-filtering: Apply the filter after the rerank step.
+
+$$search \Rightarrow rerank \Rightarrow filter \longmapsto candidates$$
+
+For a very tight (highly selective) filter, if only 1% of the rows may be selected. In this case, `pre-filtering` can reduce the number of `reranks` required by about 100 times, thus significantly improving query performance.
```

Comment:
I think the description here is quite confusing: what does `selected`, `reranks` mean?